src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+public class DeleteCommand extends Command {
+    private int index;
+
+    public DeleteCommand(int index) {
+        this.index = index;
+    }
+
+    /**
+     * Executes the command.
+     *
+     * @param taskList the TaskList used during execution.
+     * @param storage the Storage used during execution.
+     * @param ui the Ui used during execution.
+     */
+    @Override
+    public void execute(TaskList taskList, Storage storage, Ui ui) {
```

Comment:
I think perhaps you can indicate that this method throws `DukeException` by adding `throws DukeException` after the parameters.